applications/webhooks/src/modules/webhooks_parking.erl

```diff
@@ -0,0 +1,91 @@
+%%%-------------------------------------------------------------------
+%%% @copyright (C) 2015, 2600Hz INC
+%%%
+%%% @contributors
+%%% Peter Defebvre
+%%%-------------------------------------------------------------------
+
+-module(webhooks_parking).
+
+-export([init/0
+         ,bindings_and_responders/0
+         ,handle/2
+        ]).
+
+-include("../webhooks.hrl").
+
+-define(ID, wh_util:to_binary(?MODULE)).
+-define(NAME, <<"parking">>).
+-define(DESC, <<"Events when calls get parked/retrieved">>).
+-define(METADATA
+        ,wh_json:from_list([{<<"_id">>, ?ID}
+                            ,{<<"name">>, ?NAME}
+                            ,{<<"description">>, ?DESC}
+                           ])
+       ).
+
+%%--------------------------------------------------------------------
+%% @public
+%% @doc
+%% @end
+%%--------------------------------------------------------------------
+-spec init() -> 'ok'.
+init() ->
+    webhooks_util:init_metadata(?ID, ?METADATA).
+
+%%--------------------------------------------------------------------
+%% @public
+%% @doc
+%% @end
+%%--------------------------------------------------------------------
+-spec bindings_and_responders() -> {gen_listener:bindings(), gen_listener:responders()}.
+bindings_and_responders() ->
+    {[{'call', [{'restrict_to', ['PARKED', 'RETRIEVED']}
+                ,'federate'
+               ]
+      }
+     ]
+     ,[{{?MODULE, 'handle'}
+        ,[{<<"call_event">>, <<"PARKED">>}
+          ,{<<"call_event">>, <<"RETRIEVED">>}]
+       }
+      ]
+    }.
+
+%%--------------------------------------------------------------------
+%% @public
+%% @doc
+%% @end
+%%--------------------------------------------------------------------
+handle(JObj, _Props) ->
+    'true' = wapi_call:event_v(JObj),
+    AccountId = wh_json:get_value(<<"Account-ID">>, JObj),
+    maybe_send_event(AccountId, format(JObj)).
+
+%%--------------------------------------------------------------------
+%% @private
+%% @doc
+%% @end
+%%--------------------------------------------------------------------
+-spec maybe_send_event(api_binary(), wh_json:object()) -> 'ok'.
+maybe_send_event('undefined', _JObj) -> 'ok';
+maybe_send_event(AccountId, JObj) ->
+    case webhooks_util:find_webhooks(?NAME, AccountId) of
+        [] -> lager:debug("no hooks to handle ~s for ~s", [AccountId]);
```

Comment:
I thought this kind of errors would show at compile time. Maybe the lager parse transform isn't applied everywhere